### PR TITLE
Add some unit test

### PR DESCRIPTION
**Pull request by zhouhao3**
Signed-off-by: Zhou Hao <zhouhao@cn.fujitsu.com>

### CODE REVIEW

**Comment by kleimkuhler** [comment]
@zhouhao3 Could you please merge the latest `main` into this branch? We fixed some tests upstream that are failing on this branch. Thanks!

File: pkg/addr/addr_test.go [hunk 1]
```diff
@@ -198,3 +198,119 @@ func TestNetToPublic(t *testing.T) {
 		})
 	}
 }
+
+func TestParseProxyIPV4(t *testing.T) {
+	var testCeses = []struct {
```
Comment (Matei207) [inline pkg/addr/addr_test.go:203]: nit: typo in `testCeses`

File: pkg/addr/addr_test.go [hunk 2]
```diff
@@ -198,3 +198,119 @@ func TestNetToPublic(t *testing.T) {
...
+				t.Fatalf("Unexpected TCP Address: [%+v] expected: [%+v]", res, testCase.expAddr)
+			}
+
+		}
+	}
+
+}
+
+func TestProxyAddressToString(t *testing.T) {
+	var testCeses = []struct {
```
Comment (Matei207) [inline pkg/addr/addr_test.go:289]: nit: typo here also in `testCeses`

**Comment by zhouhao3** [comment]
@kleimkuhler @Matei207 updated, PTAL.

**Pull request merged**